### PR TITLE
add counter

### PR DESCRIPTION
**Pull request by rabewtf**
review my counter please

nabber coming on different branch

### CODE REVIEW

File: js/counter.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+let a = 150;
+
+count_question.addEventListener("keydown", (event) => {
+    const key = event.key;
+    if (key === "Backspace") {
+        console.log("<< --- >>")
+        if (a===150) {return;}
+        else
+        return a++;
```
Comment (Satttoshi) [inline js/counter.js:12]: You dont need the return here. 
a++;
is enough

File: js/counter.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    else { 
+        console.log("<< ++++ >>")
+    return a--;}
```
Comment (Satttoshi) [inline js/counter.js:16]: no need for return

File: js/counter.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    });
```
Comment (Satttoshi) [inline js/counter.js:17]: Keydown counts Keyboard Input, but doesn't count the amount of chars in the input field.

Better use the "Input" method and scan current amount of chars in the input field.

For inspiration check this out https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/recap-project-2/character-count-example

File: js/counter.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+let output_counter_question = document.querySelector('[data-js="output_counter_question"]');
+
+count_question.addEventListener("keyup", () =>{
+return output_counter_question.textContent=a;
```
Comment (Satttoshi) [inline js/counter.js:22]: you dont need return here.

File: js/counter.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    else { 
+        console.log("<< ++++ >>")
+    return a--;}
+    });
+
+let output_counter_answer = document.querySelector('[data-js="output_counter_answer"]');
+
+count_answer.addEventListener("keyup", () =>{
+return output_counter_answer.textContent=a;
```
Comment (Satttoshi) [inline js/counter.js:41]: you dont need return here

**Review by Satttoshi — CHANGES_REQUESTED** [review]
Your counter.js file is not behaving the way it should. Please check comments in code review.

File: js/counter.js (hunk 5 above)
Comment (rabewtf) [reply js/counter.js:41]: made it work now in the asked manner. thanks for the hints. didn't see the comments nor link to the example . 👍

**Pull request merged**